docs/quickstart/web.md

```diff
@@ -0,0 +1,53 @@
+---
+bodyclass: docs
+layout: docs
+headline: Web Quickstart
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+<p class="lead">This guide gets you started with gRPC-Web with a simple
+working example from the browser.</p>
+
+<div id="toc"></div>
+
+## Prerequisites
+
+* `docker` and `docker-compose`
+
+Please refer to [Docker website][] on how to install Docker.
```

Comment:
Is this valid markdown?

Reply:
Yes, the link is at the bottom. I tested that out.